src/Equinor.ProCoSys.IPO.Query/GetInvitationsByCommPkgNo/GetInvitationsByCommPkgNoQueryHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Fusion.Integration.Meeting;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Query.GetInvitationsByCommPkgNo
+{
+    public class GetInvitationsByCommPkgNoQueryHandler : IRequestHandler<GetInvitationsByCommPkgNoQuery, Result<List<InvitationForMainDto>>>
+    {
+        private readonly IReadOnlyContext _context;
+        private readonly IFusionMeetingClient _meetingClient;
+
+        public GetInvitationsByCommPkgNoQueryHandler(
+            IReadOnlyContext context,
+            IFusionMeetingClient meetingClient)
+        {
+            _context = context;
+            _meetingClient = meetingClient;
+        }
+
+        public async Task<Result<List<InvitationForMainDto>>> Handle(GetInvitationsByCommPkgNoQuery request,
+            CancellationToken token)
+        {
+            var invitations = await
+                (from invitation in _context.QuerySet<Invitation>()
+                    .Include(i => i.CommPkgs)
+                    .Include(i => i.McPkgs)
+                    .Where(i => i.ProjectName == request.ProjectName 
```

Comment:
If an IPO has 5 MCpkgs in the scope, will we end up getting 5 "instances" og the invitation? If so should "filter" to get only one intance of invitation per invitationId, if that makes sence?

Reply:
That makes sense, and I hadn't thought of it so really good point. I've tested it and verified that it only gives one instance of each invitation.